docker/README.md

```diff
@@ -0,0 +1,29 @@
+# Creating docker containers for snapcraft
+
+## Setup
+
+    sudo snap install docker
+
+Ensure networking works, in some cases `dockerd` might need to have something
+like `--dns=8.8.8.8` added to the command.
+
+## Creating containers
+
+Note that in the following commands, `--network host` is not strictly needed
+if a proper docker bridge is setup.
```

Comment:
I would move this right after the command that uses --network. Or maybe before, but definitely closer.

Reply:
Ah, good call.